Box.V2/Models/Request/BoxCommentRequest.cs

```diff
@@ -14,10 +14,16 @@ public class BoxCommentRequest
         public BoxRequestEntity Item { get; set; }
 
         /// <summary>
-        /// The message
+        /// The text body of the comment
         /// </summary>
         [JsonProperty(PropertyName = "message")]
         public string Message { get; set; }
 
+        /// <summary>
+        /// The text body of the comment, including @[userid:Username] somewhere in the message to mention the user, which will send them a direct email, letting them know they’ve been mentioned in a comment.
+        /// </summary>
+        [JsonProperty(PropertyName = "tagged_message")]
```

Comment:
@cburnette I checked on our public doc, the request need either message or tagged_message to be there.

I would question our API design for adding this tagged_message, which we should able to parse @[userid:Username] just from message